testing/e2e/skr/skr-binding-test/index.js

```diff
@@ -64,10 +99,38 @@ describe('SKR Binding test', function() {
     await getSecret(secretName, ns);
   });
 
+  it('Fetch SKR binding created using Gardener', async function() {
+    const resp = await keb.getBinding(options.instanceID, bindingID);
+    expect(resp.data.credentials.kubeconfig).to.equal(kubeconfigFromBinding);
+  });
+
+  it('Delete SKR binding created using Gardener', async function() {
+    const resp = await keb.deleteBinding(options.instanceID, bindingID);
+    expect(resp.status).equal(200);
+
+    try {
+      await keb.getBinding(options.instanceID, bindingID);
+      expect.fail('The call was expected to fail but it passed');
+    } catch (err) {
+      if (err.response) {
+        expect(err.response.status).equal(404);
+        console.log('Got response:');
+        console.log(err.response.data);
+      } else {
+        throw err;
+      }
+    }
+  });
+
+  it('Try to fetch sap-btp-manager secret using binding from Gardener', async function() {
```

Comment:
Why is this different then the previous "get and expect to fail" case?

Reply:
Fetch secret using kubeconfig from Gardener after binding deletion should pass because, it is not possible to invalidate admin kubeconfig.